content/2016.09.fullstack-fest-en.rst

```diff
@@ -0,0 +1,195 @@
+
+The old times where we could inspect the source code of a page ­— and learn from it — is far! It is sad, but we may have to accept it because WebAssembly goes a lot further and ships code that looks closely to assembler! It's far from the Web we know, but it's still the open Web! Note that it is very different from what we saw in the past with plugins like Java applets, now a WebAssembly function can be called from a usual JS script!
+
+The physical Web is awesome and I'm really excited about this revolution, especially how it could bring a peer-to-peer local Web, using `Web bluetooth <https://developer.mozilla.org/en-US/docs/Web/API/Web_Bluetooth_API>`_ for example. Scott told me that Mozilla approach with `FlyWeb <https://hacks.mozilla.org/2016/09/flyweb-pure-web-cross-device-interaction/>`_ is slightly different because it goes through Wifi, but both teams are in touch. The physical Web is an open project, and Google is pushing it strongly. I'm bothered by the *URL redirections* strategy though beacause it puts the redirection intermadiary in a very strong position. I believe that Google wants to be that intermediary.
+
+
+Personal feedback
+=================
+
+I never attended a conference that was so close to my flat! 3min walking! It was kind of weird to meet so many prestigious IT talents in my neighbourhood! ...and so many floppy disks :)
+
+.. image:: {filename}/images/fullstackfest-floppy.jpg
+    :alt: CC-BY @fullstackfest
+    :align: center
+
+The venue was great, and very well setup for the show (`see the introductory video clip! <https://www.youtube.com/watch?v=vxMASndC3k4&index=1&list=PLe9psSNJBf76DOOKMkDpyo_A5PfZk7JWc>`_). The coffee was not so good though.
+
+I really enjoyed the fact that there was only **one track**. I never had to divide myself between two interesting talks, and could just sit and enjoy :)
+
+Almost every topic resonated with the challenges we face at work. We are one of the few teams at Mozilla that do both backend and frontend, and that's awesome to feel in the same boat as the whole room of attendees. For example, we implemented some `Amazon lambda <https://amo2kinto-lambda.readthedocs.io>`_ recently, we are well aware of the frontend architecture challenges through our `Kinto-admin <https://github.com/Kinto/kinto-admin>`_, Ethan has a functional programming background in Haskell, and Nico and Mathieu `are playing with Elm <https://github.com/n1k0/kinto-elm-experiments>`_. That's why I strongly regretted that my teammates were not with me!
+
+Also, I had the opportunity to meet Luca and Anthony from the Kuzzle team! We had a lot of fun together and our conversations about generic reusable backends were sooner insightful — c.f. `post on kinto ML <https://mail.mozilla.org/pipermail/kinto/2016-September/000197.html>`_.
```

Comment:
"sooner" should maybe be "super"?

To me "c.f." means "compare with", especially to indicate that there's a contrast. I'd probably write "see" here, possibly with "... for more info" at the end.

"post" should probably be "my post".